src/pages/Campaign/LeaderboardLayout.tsx

```diff
@@ -22,14 +22,21 @@ import {
 } from 'state/campaigns/hooks'
 import InfoHelper from 'components/InfoHelper'
 import { CampaignState } from 'state/campaigns/actions'
+import BigNumberJS from 'bignumber.js'
```

Comment:
we also have bignumber from ether or JSBI, why do we need bignumber.js?

Reply:
Ok.